modules/rosa-configuring-aws-account.adoc

```diff
@@ -64,14 +64,7 @@ $ aws sts get-caller-identity
 .Example output
 [source,terminal]
 ----
----------------------------------------------------------------------------------
-|                                GetCallerIdentity                              |
-+-------------------------------------------------------------------------------+
-|+-----------------------------------+-----------------------+-----------------+|
-||      Account        |                Arn              |       UserID        ||
-|+-----------------------------------+-----------------------+-----------------+|
-||  <account_name>     |  arn:aws:iam<string>:user:name  |      <userID>       ||
-|+-----------------------------------+-----------------------+-----------------+|
+<aws_account_id>    arn:aws:iam::<aws_account_id>:user/<username>  <aws_user_id>
```

Comment:
Once customer set the output to table format the table still work. May I have more information why it is out of date?
`lixue@Xue-Lis-MacBook-Pro ~ % aws sts get-caller-identity
-----------------------------------------------------------------------------------
|                                GetCallerIdentity                                |
+--------------+----------------------------------------+-------------------------+
|    Account   |                  Arn                   |         UserId          |
+--------------+----------------------------------------+-------------------------+
|  301721915996|  arn:aws:iam::301721915996:user/xueli  |  AIDAJ6DYXER77CARDORK4  |
+--------------+----------------------------------------+-------------------------+`